src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/IssueDecisionSelectRecipients.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.sptribs.caseworker.event.page;
+
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.sptribs.caseworker.model.CaseIssueDecision;
+import uk.gov.hmcts.sptribs.ciccase.model.CaseData;
+import uk.gov.hmcts.sptribs.ciccase.model.State;
+import uk.gov.hmcts.sptribs.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.sptribs.common.ccd.PageBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class IssueDecisionSelectRecipients implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder
+            .page("issueDecisionSelectRecipients", this::midEvent)
+            .pageLabel("Select recipients")
+            .label("LabelRecipients","Who should receive this decision notice?")
+            .complex(CaseData::getCaseIssueDecision)
+            .optional(CaseIssueDecision::getRecipientSubject,"")
+            .optional(CaseIssueDecision::getRecipientRepresentative,"")
+            .optional(CaseIssueDecision::getRecipientRespondent,"")
+            .optional(CaseIssueDecision::getRecipientOther,"")
```

Comment:
Tamir, Other should not be displayed. 

**Please see the note as in jiira:** "Technical Note: 'Other' as shown in the wireframe is out of scope. This is because we may not have captured the contact information any other parties not identified within the case so by selecting 'Other' there will be not contact information"